src/components/sw360/index.ts

```diff
@@ -17,6 +17,7 @@ import QuickFilter from './QuickFilter/QuickFilter'
 import SearchUsersModal from './SearchUsersModal//SearchUsersModal'
 import SearchVendorsModal from './SearchVendorsModal/SearchVendorsModal'
 import SideBar from './SideBar/SideBar'
+import ToastMessage from './ToastContainer/Toast'
```

Comment:
Toast is a client component so you need to add 'use client' on top of this file or Toast.tsx. It will solve the error

Reply:
Toast.tsx it is.